### DISCUSSION TITLE
how about making people who use daily.dev more productivity?

### DISCUSSION THREAD

**Discussion by 1Anas-unknow1** [dailydotdev/daily#1484] [Feature Requests]
It's a good idea to add a page that contains the **Pomodoro timer** to avoid burnout or laziness. It is easy to code and makes a lot of difference for people who use daily.dev, it would be great if you could make a page like [Flocus by GRIDFITI ](https://app.flocus.com/) 

If that happens that would be cool, to be honest.
**Thanks for developing daily.dev**

**Comment by idoshamun** [comment]
Hi @1Anas-unknow1,

Thanks a lot for suggesting this idea. We will consider it :)
